src/main/java/org/beanpod/switchboard/aop/StreamAspect.java

```diff
@@ -76,20 +73,4 @@ public void createStream(Object result) {
     streamLogService.createLog(
         OffsetDateTime.now(), message, decoderSerial, encoderSerial, streamId.toString());
   }
-
-  @Before("execution(* org.beanpod.switchboard.controller.StreamController.deleteStream(..))")
-  public void deleteStream(JoinPoint joinPoint) {
-    Object[] args = joinPoint.getArgs();
-    Long streamId = (Long) args[0];
-    StreamDto streamDto = streamDao.getStreamById(streamId);
-    String decoderSerial = streamDto.getInputChannel().getDecoder().getSerialNumber();
-    String encoderSerial = streamDto.getOutputChannel().getEncoder().getSerialNumber();
-    String message =
-        String.format(
-            "Deleted stream of ID %d between decoder %s and encoder %s",
-            streamId, decoderSerial, encoderSerial);
-
-    streamLogService.createLog(
-        OffsetDateTime.now(), message, decoderSerial, encoderSerial, streamId.toString());
-  }
```

Comment:
I think we should re-add this, the purpose is to have a record (when getting all logs) of streams that used to exist in the system but have since been removed

Reply:
Discussed with Joshua & he explained it's risky to have an object in the database that refers to an entity that is already deleted (here, the stream will be deleted while this log will remain).